docs/2_2_common_mechanisms.adoc

```diff
@@ -893,19 +912,134 @@ include::../headers/fmi3FunctionTypes.h[tag=IntervalQualifier]
+  In <<EventMode>>, <<ClockActivationMode>> and <<ClockUpdateMode>>, the new Clock interval is always relative to the time of the current <<EventMode>>, <<ClockActivationMode>> or <<ClockUpdateMode>>.
+  In Scheduled Execution <<fmi3IntervalChanged>> means that the corresponding model partition must be scheduled or re-scheduled (if a previous call to <<fmi3GetInterval>> returned <<fmi3IntervalChanged>>).
 
 [[fmi3IntervalUnchanged,`fmi3IntervalUnchanged`]]
-  * `fmi3IntervalUnchanged` is returned if a previous call to <<fmi3GetInterval>> already returned a value qualified with <<fmi3IntervalChanged>> which has not changed since.
+  * `fmi3IntervalUnchanged` is returned if a previous call to <<fmi3GetInterval>> already returned a value qualified with <<fmi3IntervalChanged>> which has not changed since, or if the FMU is not allowed to change the interval because it is meant to be set by the importer or already fixed for the simulation (e.g., the interval of a constant Clock).
+  In this case, <<intervals-get>> returns the old unchanged interval.
   In Scheduled Execution this means the corresponding model partition has already been scheduled.
 
 [[fmi3IntervalNotYetKnown,`fmi3IntervalNotYetKnown`]]
- * `fmi3IntervalNotYetKnown` is returned for a <<countdown-aperiodic-clock,`countdown`>> Clock for which the next interval is not yet known.
- This qualifier value can only be returned directly after the Clock was active and previous calls to <<fmi3GetInterval>> never returned <<fmi3IntervalChanged>> (nor <<fmi3IntervalUnchanged>>).
- In Scheduled Execution this return value means that the corresponding model partition cannot be scheduled yet.
+  * `fmi3IntervalNotYetKnown` is returned for a <<countdown-aperiodic-clock,`countdown`>> Clock for which the next interval is not yet known.
+  This qualifier value can only be returned directly after the Clock was active and previous calls to <<fmi3GetInterval>> never returned <<fmi3IntervalChanged>> (nor <<fmi3IntervalUnchanged>>).
+  If `fmi3IntervalNotYetKnown` is returned, the corresponding value in <<intervals-get>> is not defined.
+  In Scheduled Execution this return value means that the corresponding model partition cannot be scheduled yet.
+
+<<table-qualifiers-for-time-based-clocks>> shows the allowed qualifiers for different time-based Clock types.
+For some combinations the FMU state is relevant.
+If a cell has two values, the left value refers to <<InitializationMode>>, whereas the right value refers to <<EventMode>>, <<ClockActivationMode>>, or <<ClockUpdateMode>>.
+
+The categorization of fixed and tunable Clocks as "_intervalDecimal specified_" and "_intervalDecimal not specified_" refers to the presence or absence of the <<intervalDecimal>> attribute on the Clock element in the model description.
+For a detailed explanation, refer to the definitions of <<fixed-periodic-clock,fixed Clocks>> and <<tunable-periodic-clock,tunable Clocks>>.
+
+.Allowed qualifiers for time-based Clocks.
```

Comment:
To validate this table, I would need a second brain. Can we schedule a meeting?

Reply:
> No, you are right. This is a bug/contradiction in the current specification. This needs to be fixed independently from the outcome of this PR.

@masoud-najafi , @PTaeuberDS : can you please create a separate issue / pull request for that? Thanks!

Reply:
Done, s. #2068